fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalHashJoin.java

```diff
@@ -98,6 +95,31 @@ public PhysicalHashJoin(
                 groupExpression, logicalProperties, physicalProperties, statistics, leftChild, rightChild);
     }
 
+    /**
+     * Get all used slots from hashJoinConjuncts of join.
+     * Return pair of left used slots and right used slots.
+     */
+    public Pair<List<ExprId>, List<ExprId>> getHashConjunctsExprIds() {
```

Comment:
LogicalJoin also has hashJoinConjuncts. why only PhysicalHashJoin getHashConjunctsExprIds?
In my understanding, this change disables LogicalJoin to getHashConjunctsExprIds.